### PR TITLE
DCOS-10087: Replacing warning message with toggle button tooltip

### PR DESCRIPTION
**Pull request by wavesoft**
This PR moves the JSON-only warning message from the `ServiceFormModal` header inside a tooltip in the JSON toggle button.

image

### CODE REVIEW

File: src/js/components/modals/ServiceFormModal.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ import ServiceUtil from '../../utils/ServiceUtil';
 import ToggleButton from '../ToggleButton';
 import ErrorPaths from '../../constants/ErrorPaths';
 
+const SupportedOnlyInJSONText = 'Your config contains attributes we currently only support in the JSON mode.';
```
Comment (nLight) [inline src/js/components/modals/ServiceFormModal.js]: I generally prefer SUPPORTED_ONLY_IN_JSON_TEXT for contant `const` #nit

Comment (wavesoft) [inline src/js/components/modals/ServiceFormModal.js]: You are so right 😄 Fixed!

**Comment by orlandohohmeier** [comment]
@wavesoft The tooltip looks a little strange with such a long line, can we break the text?

**Comment by wavesoft** [comment]
Done and done 👍

**Comment by wavesoft** [comment]
Yeah it looks better 👍 

image

**Comment by orlandohohmeier** [comment]
LGTM 🎆 🚢 🇮🇹

**Pull request merged**